Core/Core/AvoidingHelpers/Error/CustomError.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CustomError.swift
+//  Core
+//
+//  Created by Eugene Yatsenko on 10.10.2023.
+//
+
+import Foundation
+
+public enum CustomError: Error {
```

Comment:
Should be a more meaningful name for better understanding the area of this error.
Mb AuthError if it's' connected to the Auth logic.